instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Comment from './Comment';
+import CommentInput from './CommentInput';
+
+
+class CommentSection extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            comments: props.comments,
+            comment:'',
+            date: props.date
+            
+        };
+    }
+
+    componentDidMount() {
+        let id = this.props.postId;
+        if (localStorage.getItem(id)){
+            this.setState({
+                comments: JSON.parse(localStorage.getItem(this.props.postId))
+            });
+        }
+        else {
+            this.setComments();
+        }
+    }
+
+    componentWillUnmount(){
+        this.setComments();
+    }
+
```

Comment:
Good idea to save (set the comments in `localStorage`) upon unmounting the component.